logstash_async/database.py

```diff
@@ -96,6 +100,13 @@ def _handle_sqlite_error(self):
         _, exc, _ = sys.exc_info()
         if str(exc) == 'database is locked':
             raise DatabaseLockedError from exc
+        if str(exc) == 'disk I/O error':
```

Comment:
Did you take these message texts from real errors or is there some reference?

Reply:
@eht16 yes, these were copied from real exception messages